qualang_tools/config/helper_tools.py

```diff
@@ -0,0 +1,390 @@
+            raise KeyError(
+                f"The operation '{operation_name}' in not defined in the config for the element '{element}'."
+            )
+
+        return self.data["pulses"][
+            self.data["elements"][element]["operations"][operation_name]
+        ]
+
+    def update_op_amp(self, element: str, operation_name: str, amp: float):
+        """
+        Update the operation amplitude.
+        Can only access amplitude for a constant pulse of a single element.
+
+        :param element: name of the element to get the waveforms from. Must be defined in the config.
+        :param operation_name: name of the operation to get the waveforms from. Must be defined in the config.
+        :param amp: float for the updated amplitude. Must be within the range [-0.5, 0.5) V.
+        """
+
+        # Check inputs
+        if element not in self.data["elements"].keys():
+            raise KeyError(f"The element '{element}' in not defined in the config.")
+        if operation_name not in self.data["elements"][element]["operations"].keys():
+            raise KeyError(
+                f"The operation '{operation_name}' in not defined in the config for the element '{element}'."
+            )
+        if not -0.5 <= amp < 0.5:
+            raise ValueError(
+                "The amplitude must be within the range [-0.5, 0.5) Volts."
+            )
+
+        pulse = self.get_pulse_from_op(element, operation_name)
+        try:
+            self.data["waveforms"][pulse["waveforms"]["single"]]["sample"] = amp
+        except KeyError:
+            raise KeyError(
+                "Can only access amplitude for a constant pulse of a single element"
+            )
+
+    def get_op_amp(self, element: str, operation_name: str) -> float:
+        """
+        Get the amplitude corresponding to the given element and constant operation.
+        Can only access amplitude for a constant pulse of a single element.
+
+        :param element: name of the element to get the waveforms from. Must be defined in the config.
+        :param operation_name: name of the operation to get the waveforms from. Must be defined in the config.
+        :return: the corresponding amplitude.
+        """
+        # Check inputs
+        if element not in self.data["elements"].keys():
+            raise KeyError(f"The element '{element}' in not defined in the config.")
+        if operation_name not in self.data["elements"][element]["operations"].keys():
+            raise KeyError(
+                f"The operation '{operation_name}' in not defined in the config for the element '{element}'."
+            )
+
+        pulse = self.get_pulse_from_op(element, operation_name)
+        try:
+            return self.data["waveforms"][pulse["waveforms"]["single"]]["sample"]
+        except KeyError:
+            raise KeyError(
+                "Can only access amplitude for a constant pulse of a single element"
+            )
+
+    def update_integration_weight(
```

Comment:
same as for amplitude, although integration weights are less commonly shared

Reply:
Good point as well, I did the same as for update_op_amp